### PR TITLE
feat(widget): light and dark themes

### PR DESCRIPTION
**Pull request by JFrankfurt**
image
image

### CODE REVIEW

File: src/theme/token-details-widget-theme.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { colors } from './colors'
+
+export const LIGHT_THEME = {
```
Comment (vm) [inline src/theme/token-details-widget-theme.ts:3]: I think if you do `useTheme`, might get the dark/light theme switching for free?

Comment (JFrankfurt) [reply src/theme/token-details-widget-theme.ts:3]: That was my first thought too, but `useTheme` gives you the current context value for the `styled-components` theme, and we need the (significantly different) widget theme here only.

Comment (vm) [reply src/theme/token-details-widget-theme.ts:3]: ah okay, np

Comment (vm) [reply src/theme/token-details-widget-theme.ts:3]: similar comment around whether this is token-details specific as below

File: src/utils/token-details-widget-config.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { SupportedChainId } from 'constants/chains'
```
[the remark this answers is not in the document]
Comment (JFrankfurt) [reply src/utils/token-details-widget-config.ts:1]: **distant** future we might replace the /swap interface with this

**Review by vm — APPROVED** [review]
personally recommend renaming to rm token-details, but looks good to me!

**Pull request merged**